src/sentry/mediators/sentry_app_installations/destroyer.py

```diff
@@ -18,5 +21,15 @@ def _destroy_authorization(self):
     def _destroy_grant(self):
         self.install.api_grant.delete()
 
+    def _destroy_service_hooks(self):
+        hooks = ServiceHook.objects.filter(
+            application=self.install.sentry_app.application,
+            actor_id=self.install.id,
+        )
```

Comment:
Might be useful to add a `SentryAppInstallation#service_hooks` method that exposes these. Not a blocker – just a thought.